README.rst

```diff
@@ -69,15 +69,15 @@ Afterwards Bob spends 3 of these tokens.
 
     # import BigchainDB and create an object
     from bigchaindb_driver import BigchainDB
-    bdb_root_url = 'https://example.com:9984'
+    bdb_root_url = 'http://localhost:32804'
```

Comment:
Why did you change the port to `32804`?

Reply:
ah, shoot.. I reverted it twice and it got back to my test settings. I'll change it to the original